src/app.js

```diff
@@ -80,7 +79,14 @@ function validateResponseAndStatusCode(response){
  */
 function callApi(event, query, context) {
   return validateAndExecuteTransform(event, query, context).then(response => {
-    if (validateResponseAndStatusCode(response)) {
+    const validateErrors = validateResponse(event, response.data);
+    if (validateErrors.length) {
+      return {
+        statusCode: 400,
```

Comment:
Jeg vil foreslå at vi kun logger en warning internt, men **ikke** fejler hvis responset ikke validerer.

Eksempelvis er der vistnok poster i brønden med fejl i inddateringen af feltnavne, der vel i så fald vil fejle valideringen op i mod `Work`, hvis en af disser poster er med i søgeresultatet. Her vil vi nok hellere have at resultatet alligevel bliver leveret til klienten, end at fejle, men vi vil gerne vide det i loggen.

Reply:
Som udgangspunkt fejles der ikke, når der er ekstra felter med, kun når de specificerede felter har forkert format. Men derudover er jeg enig i at det ikke giver så meget værdi for brugeren at servicen fejler, da de ikke kan handle på det. Jeg opretter et issue på det, da det kræver en diskussion i teamet og med med DDB